lib/fluent/plugin/exception_detector.rb

```diff
@@ -61,7 +61,8 @@ def self.supported
       rule([:java_after_exception, :java], /^[\t ]+(?:eval )?at /, :java),
       rule([:java_after_exception, :java], /^[\t ]*(?:Caused by|Suppressed):/,
            :java_after_exception),
-      rule([:java_after_exception, :java], /^[\t ]*... \d+\ more/, :java)
+      rule([:java_after_exception, :java],
+           /^[\t ]*... \d+ (?:more|common frames omitted)$/, :java)
```

Comment:
why adding `$` suddenly? and I cannot verify whether java would not add `'\n'`, or other characters after those text. Do you have reference on source code?

Reply:
Good point. The text ("more" or "common frames omitted") does appear at the end of the line, but log messages may also have linefeed/newline characters appended, so it's better to be careful. Fixed.